content/docs/alerting/configuration.md

```diff
@@ -128,6 +128,17 @@ current node.
 
 ```yaml
 [ receiver: <string> ]
+# The labels by which incoming alerts are grouped together. For example,
+# multiple alerts coming in for cluster=A and alertname=LatencyHigh would
+# be batched into a single group.
+#
+# To aggregate by all possible labels use the special value '...' as the sole label name:
```

Comment:
You can combine this and the next line,  

...name for example:

Reply:
done